FirebaseTestProject/ViewModels/SignInViewModel.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  SignInViewModel.swift
+//  SignInViewModel
+//
+//  Created by Tim Miller on 8/12/21.
+//
+
+
+import FirebaseAuth
+
+class SignInViewModel: ObservableObject {
```

Comment:
I'm not sure what the best name is for this view model, but `SignInViewModel` feels a bit clunky

Reply:
renamed to AuthViewModel?